### ISSUE TITLE
util/atomic.h and avr/interupt.h are missing

### ISSUE THREAD

**Issue by rosahas** [cyborg5/IRLib2#16]
I keep getting errors that these files are missing, both on Win and Mac.

**Comment by cyborg5** [comment]
Those files should have installed with the Arduino IDE. They are part of the standard Arduino infrastructure. Do you have the latest IDE installed?

**Comment by rosahas** [comment]
Yes, I have the latest IDE, but it's on a Mac.  I am guessing they are not found.  Perhaps it will work on Windows.
